rules-java-archives/addon/src/main/java/org/jboss/windup/rules/apps/java/archives/listener/ArchiveIdentificationGraphChangedListener.java

```diff
@@ -21,43 +21,49 @@
 import com.tinkerpop.blueprints.Edge;
 import com.tinkerpop.blueprints.Vertex;
 import com.tinkerpop.blueprints.util.wrappers.event.listener.GraphChangedListener;
+import org.jboss.windup.util.Logging;
 
 /**
  * {@link GraphChangedListener} responsible for identifying {@link ArchiveModel} instances when they are added to the graph.
  *
  * @author <a href="mailto:lincolnbaxter@gmail.com">Lincoln Baxter, III</a>
  * @author <a href="mailto:ozizka@redhat.com">Ondrej Zizka</a>
  */
-public class ArchiveIdentificationGraphChangedListener implements GraphChangedListener
+public final class ArchiveIdentificationGraphChangedListener implements GraphChangedListener
 {
-    private static final Logger log = Logger.getLogger(ArchiveIdentificationGraphChangedListener.class.getName());
+    private static final Logger LOG = Logging.get(ArchiveIdentificationGraphChangedListener.class);
 
     private final ArchiveIdentificationService identifier;
-
     private GraphContext context;
+    private ArchiveService archiveService;
+
+    public ArchiveIdentificationGraphChangedListener setGraphContext(GraphContext context)
```

Comment:
Sometimes the might want to set the graph context? Otherwise I think no reason rather than IDE generating it that way by default.